test/cmd/export.sh

```diff
@@ -8,6 +8,8 @@ set -o pipefail
 
 oc new-app -f examples/sample-app/application-template-stibuild.json --name=sample
 
+oc export all --all --all-namespaces --exact
```

Comment:
it seems silly that the "all" alias doesn't impicitly also mean "--all"

I guess i could in theory want to say "oc export all things_named_foo" but is that really common?